sycl/doc/extensions/supported/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md

```diff
@@ -12,8 +55,15 @@ Explicit SIMD provides the following key features complementary to SYCL:
   general register file. This allows to write efficient code not relying on
   further widening by the compiler, as with traditional SPMD programming.
 - Low-level APIs efficiently mapped to the Intel GPU architecture, such as block loads/stores/gathers/scatters, explicit cache hints, GPU inline assembly, etc.
+- Regular SYCL and ESIMD kernels can co-exist in the same translation unit and in
+the same application. See more details in ./README.md#
```

Comment:
```suggestion
the same application. For more details, see ./README.md#
```

Reply:
Nick, Thank you.
I've done fixes for this and other comments.
This line definitely deserved a fix because it had a broken link, which is fixed now.

BTW, SYCL-ESIMD-Interoperability section probably should be moved from README to the main ESIMD SPEC. I can do that a bit later.

Reply:
cool thx